controlplane/kubeadm/internal/machinefilters/util_test.go

```diff
@@ -0,0 +1,825 @@
+/*
```

Comment:
What's up with `util_test.go` rather than `machine_filters_test.go` for testing `machine_filters.go`?

Reply:
that's a tricky one.
`machine_filters_test` currently uses a different namespace name, machine_filters_test, and it can access private func in machine_filters. If I try to fix the namespace name, then I got into a circular reference problem due to other imports in the test.
So I create another test file to keep moving.